<!-- Header -->
<launch>
 
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find ar_demo)/rosconsole.conf"/>

  <rosparam command="load" file="$(find ar_demo)/parameters/parameters.yaml" />


 
  <!--
  ++++ Camera related nodes
  -->
 
<node pkg="ar_demo" type="camera_node" name="camera_node" args="&#x002d;&#x002d;camera aca2500_20gc_21768100" respawn="false" required="true" output="screen" />

  <node pkg="ar_demo" type="camera_display_node" name="camera_display_node" args="&#x002d;&#x002d;camera aca2500_20gc_21768100 &#x002d;&#x002d;downsample 4" respawn="false" required="true" output="screen" />

<!-- Intrinsics calibration -->

<node pkg="ar_demo" type="intrinsics_calib.py" name="intrinsics_calibration_tool_node" args="aca2500_20gc_21768100" respawn="false" required="true" output="screen" />


</launch>
